backend/question-service/src/controllers/questionController.ts

```diff
@@ -85,6 +89,92 @@ export const updateQuestion = async (
       question: updatedQuestion,
     });
   } catch (error) {
-    res.status(500).json({ message: "Server error", error });
+    res.status(500).json({ message: SERVER_ERROR, error });
+  }
+};
+
+export const readQuestionsList = async (
+  req: Request,
+  res: Response,
+): Promise<void> => {
+  try {
+    const qnLimit = 10;
```

Comment:
i think it would be better if we use query params to specify the number of items per page and how many items to skip? instead of fixing it to 10. so maybe something like `const {limit, skip} = req.query`

Reply:
i used 10 bcuz we wrote that in our fr

Reply:
oh, then we can still use the query params right? just specify 10 in the frontend?

Reply:
oh okay sure